contracts/cairo/.env.example

```diff
@@ -0,0 +1,9 @@
+STARKNET_ACCOUNT=<starkli_account_path>
+STARKNET_KEYSTORE=<starkli_keystore_path>
+SN_RPC_URL=<starknet_rpc_url>
+ETH_CONTRACT_ADDR=<ethereum_contract_address> #in lowercase hexa with the 0x prefix
```

Comment:
It does not be in lowercase. Ie 0x5Aa457Fbb4A4E4FdE5011bd8dbD486961238604A. It should be as is from etherscan

Reply:
As it is a hexa, 'a' and 'A' are both '10' , right?
I clarified "in lowercase" to remain in the safe side , but I couldn't execute a case in which uppercase or lowercase makes a difference, for any of the addresses. Have you?
https://ethereum.stackexchange.com/questions/2045/is-ethereum-wallet-address-case-sensitive
Do we use this somewhere?

Reply:
> As it is a hexa, 'a' and 'A' are both '10' , right?
Yes
>  I couldn't execute a case in which uppercase or lowercase makes a difference, for any of the addresses. Have you?
web3.py fails when the address is not exactly as etherscan shows it. Maybe in contract deploys it does not matter